charts/govuk-sli-collector/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for govuk-sli-collector.
+
+govukEnvironment: test
+
+enabled: false
+intervalMinutes: "10"
+offsetMinutes: "10"
+
+image:
+  repository: "govuk-sli-collector"
```

Comment:
I copied this repository name over from the `govuk-jobs` values, but in this new chart will it need to be `"172025368201.dkr.ecr.eu-west-1.amazonaws.com/govuk-sli-collector"` instead?